packages/loopring_v3/test/types.ts

```diff
@@ -235,7 +235,7 @@ export interface TxBlock {
 export interface Block {
   blockIdx: number;
   filename: string;
-  blockType: BlockType;
+  blockType: number;
```

Comment:
@Brechtpd we still have this 'blockType', shall we remove it?

Reply:
Would keep it for now, could still make sense.